cmake/utilities.cmake

```diff
@@ -0,0 +1,92 @@
+##############################################################################################
```

Comment:
This code is under LGPL license.
It is not compliant with APACHE 2.0 license.
I cannot integrate it.
Can we remove it or any other solution?

Reply:
This macro has nothing interesting we can just remove the copyright